static/app/views/dashboardsV2/widgetCard/widgetCard.spec.tsx

```diff
@@ -435,29 +435,31 @@ describe('Dashboards > WidgetCard', function () {
 
   it('calls eventsV2 with a limit of 20 items', async function () {
     const mock = jest.fn();
-    render(
-      <WidgetCard
-        api={api}
-        organization={organization}
-        widget={{
-          ...multipleQueryWidget,
-          displayType: DisplayType.TABLE,
-          queries: [{...multipleQueryWidget.queries[0], fields: ['count()']}],
-        }}
-        selection={selection}
-        isEditing={false}
-        onDelete={mock}
-        onEdit={() => undefined}
-        onDuplicate={() => undefined}
-        renderErrorMessage={() => undefined}
-        isSorting={false}
-        currentWidgetDragging={false}
-        showContextMenu
-        widgetLimitReached={false}
-        tableItemLimit={20}
-      />
-    );
-    await tick();
+    await act(async () => {
+      render(
+        <WidgetCard
+          api={api}
+          organization={organization}
+          widget={{
+            ...multipleQueryWidget,
+            displayType: DisplayType.TABLE,
+            queries: [{...multipleQueryWidget.queries[0], fields: ['count()']}],
+          }}
+          selection={selection}
+          isEditing={false}
+          onDelete={mock}
+          onEdit={() => undefined}
+          onDuplicate={() => undefined}
+          renderErrorMessage={() => undefined}
+          isSorting={false}
+          currentWidgetDragging={false}
+          showContextMenu
+          widgetLimitReached={false}
+          tableItemLimit={20}
+        />
+      );
+      await tick();
+    });
```

Comment:
iirc state wasn't fully stabilized after render. Most of these places were complaining `act` and most places required a tick because state was being updated during mount.